internal/uvm/scsi.go

```diff
@@ -17,6 +18,26 @@ const (
 	lcowSCSILayerFmt = "/run/layers/S%d/%d"
 )
 
+// VMAccessType is used to determine the various types of access we can
+// grant for a given file.
+//
+// This can be `VMAccessTypeNoop` where no additional access is given. Note
+// this should be used for layers and gpu vhd where we have given VM group
+// access outside of the shim (containerd for layers, package installation
+// for gpu vhd).
+//
+// `VMAccessTypeGroup` if we should give access to a file for the VM group sid
+//
+// `VMAccessTypeIndividual` if we should give additional access to a file for
+// the running VM only
+type VMAccessType int
+
+const (
+	VMAccessTypeNoop VMAccessType = iota
```

Comment:
I think the doc will show up better if you move the description of each enum option to be at the declaration.

e.g.:

```
// VMAccessType is a blah blah...
type VMAccessType int

const (
    // VMAccessTypeNoop is a foo bar...
    VMAccessTypeNoop VMAccessType = iota
    // VMAccessTypeGroup is a baz quux...
    VMAccessTypeGroup
```

Reply:
fixed.